packages/requisitions/src/RequestRequisition/api/api.ts

```diff
@@ -15,20 +16,32 @@ import {
 
 export type RequestRequisitionApi = ReturnType<typeof getSdk>;
 
-export const requisitionToInput = (
-  requisition: Partial<RequestRequisitionFragment> & { id: string }
-): UpdateRequestRequisitionInput => {
-  return {
-    id: requisition.id,
-    // otherPartyId: requisition.otherParty?.id,
-    comment: requisition.comment,
-    theirReference: requisition.theirReference,
-    colour: requisition.colour,
-    status:
-      requisition.status === RequisitionNodeStatus.Sent
-        ? UpdateRequestRequisitionStatusInput.Sent
-        : undefined,
-  };
+const requisitionParser = {
+  toInsertLineInput: (line: DraftRequestRequisitionLine) => ({
+    id: line.id,
+    itemId: line.itemId,
+    requisitionId: line.requisitionId,
+    requestedQuantity: line.requestedQuantity,
+  }),
+  toUpdateLineInput: (line: DraftRequestRequisitionLine) => ({
+    id: line.id,
+    requestedQuantity: line.requestedQuantity,
+  }),
+  toInsertInput: (
+    requisition: Partial<RequestRequisitionFragment> & { id: string }
+  ): UpdateRequestRequisitionInput => {
+    return {
+      id: requisition.id,
+      // otherPartyId: requisition.otherParty?.id,
+      comment: requisition.comment,
+      theirReference: requisition.theirReference,
+      colour: requisition.colour,
+      status:
```

Comment:
not part of the insert input. Should this be `toUpdateInput`?

Reply:
Yup, thanks nice spottin!